public/static/docs/command-reference/gc.md

```diff
@@ -42,16 +47,20 @@ restored using `dvc fetch`, as long as they have previously been uploaded with
   useful if tags are used to track "checkpoints" of an experiment or project.
   Note that both options can be combined, for example using the `-aT` flag.
 
+- `-w`, `--workspace` - remove files in local cache that are not referenced in
+  the workspace. _This operation is dangerous._ If `--all-tags` or
+  `--all-branches` are specified, this flag is not explicitly required.
```

Comment:
No, they can work with `-w` or `-c`, so, it kind-of makes sense to also mention there. Also, `-a` or `-T` by default activates -`w`.

Reply:
Are we sure? Can you answer https://github.com/iterative/dvc.org/pull/1023#issuecomment-593587976 please? Copied here:

> Question: do -a and -T keep also cached files referenced in the workspace regardless of whether you're currently checked out in a branch or tag?

I find this mention about -aT here pretty confusing so we need to double check that to figure out the best phrasing (or remove). At this time I think moving this option to the top is probably enough (no mention about -aT, but probably mention -w somehow in -aT.

Reply:
Yes, when  `-a` and `-T` are provided, it is same as `-wa` and `-wT` respectively, so, it still keeps everything from workspace.

> Question: do -a and -T keep also cached files referenced in the workspace regardless of whether you're currently checked out in a branch or tag?

Yes, they should (but, I'd verify it tomorrow and update).

Reply:
> when -a and -T are provided, it is same as -wa and -wT respectively

OK, I just double checked (DVC 0.87.0) when the worskspace is in a detached head (no branch or tag) and indeed this seems to be the case. In fact `dvc gc -a` warns: `WARNING: This will remove all cache except items used in the working tree and all git branches of the current repo.`

so to summarize, my suggestion is to move -w to the top of the options list, and something like
```suggestion
  the workspace. **This behavior is dangerous.** This option is
  enabled automatically if `--all-tags` or `--all-branches` are
  used.
```
Thanks!

> I may open a separate suggestion about how to mention -w in -aT.